backend/tests/sql_export_test.go

```diff
@@ -61,8 +61,8 @@ func TestSQLExport(t *testing.T) {
 							statement: "SELECT * FROM Test1.tbl;",
 							content: `[
   {
-    "gender": "0b00000000",
```

Comment:
This will be a broken change for some customers, background https://linear.app/bytebase/issue/BYT-8096/boolean-values-must-not-use-binary-literal-format-in-json-file
cc @changyuwong